.kitchen.yml

```diff
@@ -82,6 +82,8 @@ platforms:
   - name: windows-2008r2-standard
     driver:
       box: chef/windows-server-2008r2-standard # private
+      synced_folders:
+        - ['..', '/home/vagrant']
```

Comment:
This is presumably to copy all the directories that are peers to the one you're in?  Windows has no /home - did you mean to land it in C:/Users/vagrant/repo or something like that?